controllers/suite_test.go

```diff
@@ -186,5 +191,16 @@ func createTestClusterReconciler() *FoundationDBClusterReconciler {
 		PodLifecycleManager:    podmanager.StandardPodLifecycleManager{},
 		PodClientProvider:      internal.NewMockFdbPodClient,
 		DatabaseClientProvider: mockDatabaseClientProvider{},
+		RestClient:             &fake.RESTClient{Resp: &http.Response{StatusCode: http.StatusOK}},
+		RestConfig: &rest.Config{
+			Host: "https://notused.co.uk:443",
```

Comment:
In general, I would recommend that we use a `.example` domain for a URL that we never intend to be called, so that we don't rely on unspecified behavior like a domain not existing at the moment.